src/index.html

```diff
@@ -18,5 +18,86 @@
   </head>
   <body>
     <h1>Moyo header</h1>
+    <header>
+      <a
+        href="index.html"
+        class="logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="moyoLogo"
+        />
+      </a>
+
+      <nav>
+        <ul class="nav-list">
+          <li class="nav-item">
+            <a
+              href="#"
+              class="nav-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+          <li class="nav-item">
```

Comment:
Add the blank line between the links 
```suggestion
          </li>
          
          <li class="nav-item">
```